modules/installation-obtaining-installer.adoc

```diff
@@ -138,9 +121,17 @@ Using a {cluster-manager-url-pull} is not required. You can use a pull secret fo
 +
 If you do not use the {cluster-manager-url-pull}:
 +
-* Red Hat Operators are not available.
-* The Telemetry and Insights operators do not send data to Red Hat.
-* Content from the link:https://catalog.redhat.com/software/containers/explore[Red Hat Ecosystem Catalog Container images] registry, such as image streams and Operators, are not available.
+* Red{nbsp}Hat Operators are not available.
+* The Telemetry and Insights operators do not send data to Red{nbsp}Hat.
+* Content from the link:https://catalog.redhat.com/software/containers/explore[Red{nbsp}Hat Ecosystem Catalog Container images] registry, such as image streams and Operators, are not available.
+endif::openshift-origin[]
+
+ifndef::openshift-origin[]
+[TIP]
+====
+Alternatively, you can retrieve the installation program from the link:https://access.redhat.com/downloads/content/290/[Red{nbsp}Hat Customer Portal], where you can specify a version of the installation program to download.
+However, you must have an active subscription to access this page.
```

Comment:
```suggestion
Alternatively, you can retrieve the installation program from the link:https://access.redhat.com/downloads/content/290/[Red{nbsp}Hat Customer Portal], where you can specify a version of the installation program to download. However, you must have an active subscription to access this page.
```

Reply:
Was the break here intentional? I was thinking maybe it isn't necessary, but up to you 🤷

Reply:
Yes it's intentional, however without a blank link in between, it renders the same as if both sentences were on the same line.

I do 1 sentence per line ever since someone at Red Hat introduced this style of writing, because it both helps you compare the lengths of your sentences more easily, but it also helps reviewers point out a specific sentence just by commenting on the line that has nothing but that sentence.

Reply:
Oh yea I know what you're talking about. I don't subscribe to that, but whatever fits your fancy. I will keep this in mind when reviewing other PRs for you :)